pkg/hs/natpmp-static/cbits/binding.c

```diff
@@ -54,7 +54,8 @@ int readNatResponseSynchronously(natpmp_t* natpmp, natpmpresp_t * response)
 		getnatpmprequesttimeout(natpmp, &timeout);
 		r = select(FD_SETSIZE, &fds, NULL, NULL, &timeout);
 		if(r<0) {
-			fprintf(stderr, "select()");
+			fprintf(stderr, "select():  errno=%d '%s'\n",
+			        sav_errno, strerror(sav_errno));
```

Comment:
Looks like you'll also need `sav_errno = errno;` after the `select()` call. i'm not sure if that would be better in the error-condition block, or before it.

Reply:
Ooops.